lib/run.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './input'
+require_relative './prompt'
+
+def prompt_players
+    puts Prompt::WELCOME
+end
+
+def get_players
+    input = Input.new
+    players = input.get_input
+end
+
+def prompt_token_selection
+    puts Prompt::MAKE_TOKEN_SELECTION
+end
+
+def get_player_token
+    input = Input.new
+    player_token = input.get_input
```

Comment:
Why's this `player_token` variable being assigned here?

Reply:
I want to obtain tokens before starting the game loop, but my goal is to save this input in a HumanPlayer object.

Reply:
Step one: remove assignments, done! I can worry about saving input when I need to.